cves/kernel/CVE-2019-12378.yml

```diff
@@ -149,10 +147,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    There is not a lot of information about this vulnerability, but it seems
+    that a developer, not from google, located it.
+  automated: false
+  contest: false
+  developer: true
```

Comment:
I'd be iffy on calling them a developer.  Check if they're a developer for Linux (like an actual product dev) -- if not, I wouldn't include it.

Reply:
Updated